### PR TITLE
[13.0][ADD] new module sale_stock_analytic

### PR DESCRIPTION
**Pull request by baimont**
New module that copies the analytic account of the sale order and the analytic tags of the sale order line to the stock move

Depends on https://github.com/OCA/account-analytic/pull/332

### CODE REVIEW

**Comment by sanrav** [comment]
Functional testing OK

**Review by apineux — APPROVED** [review]

**Review by sanrav — APPROVED** [review]
Functional testing OK

File: sale_stock_analytic/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "summary": """
+        Copies the analytic account of the sale order and the analytic tags
+        of the sale order line to the stock move""",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/account-analytic",
+    "depends": ["sale", "stock_analytic"],
+    "data": [],
+    "demo": [],
```
Comment (MeetKD) [inline sale_stock_analytic/__manifest__.py]: @baimont Can you please remove the empty keys ?

File: sale_stock_analytic/models/stock_rule.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            product_id,
+            product_qty,
+            product_uom,
+            location_id,
+            name,
+            origin,
+            company_id,
+            values,
+        )
+        sol_id = move_values.get("sale_line_id")
```
Comment (MeetKD) [inline sale_stock_analytic/models/stock_rule.py]:
```suggestion
        sol_id = move_values.get("sale_line_id", False)
```

File: sale_stock_analytic/models/stock_rule.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        if sol_id:
+            sol_model = self.env["sale.order.line"]
+            sol = sol_model.browse(sol_id)
+            at = sol.analytic_tag_ids
+            aa = sol.order_id.analytic_account_id
+            if at:
+                move_values.update({"analytic_tag_ids": [(6, 0, at.ids)]})
+            if aa:
```
Comment (MeetKD) [inline sale_stock_analytic/models/stock_rule.py]: @baimont Can you please also improve the variable names ?

**Review by MeetKD — CHANGES_REQUESTED** [review]
Minor Changes !

**Comment by rousseldenis** [comment]
@baimont Same remarks as for purchase module https://github.com/OCA/account-analytic/pull/411

**Review by celm1990-sh — APPROVED** [review]

**Comment by celm1990-sh** [comment]
Hi, i see PR related are merge, because this PR is nor merge yet???
@baimont @rousseldenis

**Comment by rafaelbn** [comment]
Hello @baimont , this PR is pending because you have a comment of  @rousseldenis here: https://github.com/OCA/account-analytic/pull/334#issuecomment-957345115 without answer. Thank you! 😄

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**